lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -38,9 +38,14 @@
 #include "llvm/Support/Casting.h"
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Debug.h"
+#include "llvm/Support/FileSystem.h"
 #include "llvm/Support/Format.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include <fcntl.h>
```

Comment:
Are all of these headers supported on Windows as well?

Reply:
Good point. Might not be without some sort of POSIX wrapper (I haven't developed in Windows in ages). I better rewrite these using C++ streams or the LLVM wrappers.